ideas/168-paid-master-nodes.md

```diff
@@ -0,0 +1,192 @@
+
+Solving the initial user stories don't require any proof of message delivery or
+quality of service. Reputation of specific mail servers thus happens completely
+outside of the app, e.g. through Riot, Reddit, etc.
+
+#### Note on Status Desktop
+
+Status Desktop isn't a requirement for this idea as a user can run a mail server
+from the command line. It is an additional user story that shouldn't be too
+difficult to implement, but it isn't clear how desirable it is, assuming that
+high availabity of mail servers is a requirement.
+
+#### Terminology
+
+Mail server / wnode / master node are used interchangably. Master mode is the
+"soft" user term (invented by the community!), mail server and wnode are
+technical terms on status-go side.
+
+#### Relevant reading
+
+- https://status.im/whitepaper.pdf
+- http://nakamotoinstitute.org/static/docs/micropayments-and-mental-transaction-costs.pdf
+
+### Requirements & Dependencies
+
+(Lack of consensus on this point: Possibly onboarding SNT give-away logistics.)
+
+### Minimum Viable Product
+<!-- Mandatory, completes the Idea in the fastest route possible, can be hacky, needed to feel progress. See https://imgur.com/a/HVlw3 -->
+Goal Date: TBD - May 1 + 2w
+
+Description: Spike out most basic proof of concept possible for payment / service on or off in Clojure and Go, using SNT test token.
+
+## Dates
+Separate tracks:
+- UX(R) / design
+- Go dealing with app protocol / payment
+
+Goal Date: <!-- Date for evaluation in ISO 8601 (yyyy-mm-dd) format --> 
+
+Description: <!-- Description of Deliverables-->
+
+Testing Days required: <!-- Days required at the end of development for testing -->
+
+## Success Metrics
+
+- Users running mail servers and transacting. Target: 10
+
+- Users paying for a mail server. Target: 50% of D7 retained users, or 20% of all users.
+
+- Daily Transacting Users. Target: 35 ((5000 DAU TARGET*0.2 TX%)/30 PAY-MONTHLY).
+
+OKRs:
+2.4: >20% of users send a transaction (P1:P3)
+3.1: 2x launched SNT use cases (P2:P0)
+
+## Exit criteria
+
+Solving the two initial user stories. If the swarm deems it useful to continue,
+new specific user stories will be specified and inform the exit criteria.
+
+## Supporting Role Communication
+
+- Community for recruiting users to run mail server.
+
+- Finance for SNT utility approval.
+
+- Marketing for 'you are not the product' rationale communication.
+
+## Appendix: Constraints / objections
+
+- No one will pay for it.
+
+This is a free resource. If we want to subsidise it and the concern is people
+  not wanting to stay / top up to use basic funcrtionality we can solve it by
+  giving free SNT as part of SNT onboarding.
+  
+- Free SNT?!
+
+Yes. For a closed beta limited to 40k people, we can limit the free amount to 5
+SNT per person, which limits the downside risk this to 200k SNT ($20k at the
+time of this writing).
+
+- What about fraud?
+
+We can use already collected emails as identity and dedup by it. Max downside
```

Comment:
We'll probably want to blacklist some proxy email domains such as maskr.me, 33mail.com, etc. which allow you to create unlimited email accounts.